### ISSUE TITLE
Add url change feature for some extensions because they are often blocked by network operators (XXManhwa, TruyenVn, SayHentai)

### ISSUE THREAD

**Issue by tuanbmt4152** [keiyoushi/tachiyomi-extensions-source#5562]
### Source name

XXManhwa, TruyenVn, SayHentai

### Source language

Vietnamese-Vi

### Describe your suggested feature

Change the url domain name option in the application because in our country these "unlicensed" websites are often blocked by the national network (including anime, movies) so the domain name must be changed many times.
-XXManhwa [#5379](https://github.com/keiyoushi/extensions-source/issues/5379) New domain name has been changed.
-TruyenVn [#5392](https://github.com/keiyoushi/extensions-source/issues/5392) This problem has just been fixed and after 3 days the network operator has blocked and changed the new domain name (truyenvn.wiki) so when accessing the old domain name, it will redirect to the new link.
-SayHentai Will change Url in future.

### Other details

Example of application below (lxhentai)
![462552430_470676279352999_4846042804762062519_n](https://github.com/user-attachments/assets/94fef0c5-2d69-40b2-b4c5-e9b595c6227b)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/mihonapp/mihon/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by takahashikajima** [comment]
Bạn ơi làm sao để hiện cái đổi link web này vậy chỉ mình với

**Comment by tuanbmt4152** [comment]
> Bạn ơi làm sao để hiện cái đổi link web này chỉ với mình

vào mục các tiện ích mở rộng , ấn vào hình bắnh răng cạnh app ,tiếp đến lại ấn vào bánh răng canh chữ tiếng việt là thấy Dòng là ghi đè URL
